include/databroker/DataBrokerConfiguration.hpp

```diff
@@ -45,8 +46,13 @@ struct DataBrokerParticipantConfiguration
 struct DataBrokerLocalParticipantConfiguration : public DataBrokerParticipantConfiguration
 {
     bool ros;
+    bool discovery_server;
+    eprosima::fastrtps::rtps::DiscoveryProtocol discovery_protocol;
+    std::vector<Address> listening_addresses;
+    eprosima::fastrtps::rtps::GuidPrefix_t server_guid;
```

Comment:
```suggestion
    std::vector<Address> listening_addresses;
```
I would erase those as they are not parameters, but are always the same value depending on `discovery_server`